From d58cbc6f208a2c4ab60d707db03f3f5b287ea901 Mon Sep 17 00:00:00 2001
From: Brendan King <Brendan.King@imgtec.com>
Date: Mon, 14 Dec 2015 13:32:41 +0000
Subject: [PATCH 1/3] Define K_OFF if it isn't defined already.

Define K_OFF if it isn't defined in the Linux header files used for
compilation.
---
 libweston/launcher-direct.c | 4 ++++
 libweston/weston-launch.c   | 4 ++++
 2 files changed, 8 insertions(+)

diff --git a/libweston/launcher-direct.c b/libweston/launcher-direct.c
index 9fa329b6..6e5de655 100644
--- a/libweston/launcher-direct.c
+++ b/libweston/launcher-direct.c
@@ -47,6 +47,10 @@
 #define KDSKBMUTE	0x4B51
 #endif
 
+#ifndef K_OFF
+#define K_OFF		0x4
+#endif
+
 /* major()/minor() */
 #ifdef MAJOR_IN_MKDEV
 #include <sys/mkdev.h>
diff --git a/libweston/weston-launch.c b/libweston/weston-launch.c
index 8a711b4a..0e2045c9 100644
--- a/libweston/weston-launch.c
+++ b/libweston/weston-launch.c
@@ -65,6 +65,10 @@
 #define KDSKBMUTE	0x4B51
 #endif
 
+#ifndef K_OFF
+#define K_OFF		0x4
+#endif
+
 #ifndef EVIOCREVOKE
 #define EVIOCREVOKE _IOW('E', 0x91, int)
 #endif
-- 
2.17.1

